### PR TITLE
Ecm-135 hearings to judgments

### PR DESCRIPTION
**Pull request by Angela-LW**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ECM-135

### Change description ###
add models and client end point for hearings to judgments report

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by jthmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/ecm/common/model/reports/hearingstojudgments/CaseData.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    private String ethosCaseReference;
+
+    @JsonProperty("hearingCollection")
+    private List<HearingTypeItem> hearingCollection;
+
+    @JsonProperty("judgementCollection")
+    private List<JudgementTypeItem> judgementCollection;
+
+    @JsonProperty("managingOffice")
+    private String managingOffice;
```
Comment (gauravchawla73463) [inline src/main/java/uk/gov/hmcts/ecm/common/model/reports/hearingstojudgments/CaseData.java:22]: May be this class name can be changed to something report specific to clearly differentiate the generic CaseData?

File: src/main/java/uk/gov/hmcts/ecm/common/model/reports/hearingstojudgments/HearingsToJudgmentsSearchResult.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class HearingsToJudgmentsSearchResult {
+    private Long total;
```
Comment (gauravchawla73463) [inline src/main/java/uk/gov/hmcts/ecm/common/model/reports/hearingstojudgments/HearingsToJudgmentsSearchResult.java:13]: Is this class used in ecm-consumer as well? If not, may be adding it in ethos service makes more sense?

File: src/main/java/uk/gov/hmcts/ecm/common/model/reports/hearingstojudgments/CaseData.java (hunk 1 above)
Comment (Angela-LW) [reply src/main/java/uk/gov/hmcts/ecm/common/model/reports/hearingstojudgments/CaseData.java]: renamed hearings to judgments case data object to be more distinct

File: src/main/java/uk/gov/hmcts/ecm/common/model/reports/hearingstojudgments/HearingsToJudgmentsSearchResult.java (hunk 2 above)
Comment (Angela-LW) [reply src/main/java/uk/gov/hmcts/ecm/common/model/reports/hearingstojudgments/HearingsToJudgmentsSearchResult.java:13]: this is used as part of the report elastic search method in the ccd client so can't be moved to ethos service

**Review by gauravchawla73463 — APPROVED** [review]

**Pull request merged**